lib/xenops_server_plugin.ml

```diff
@@ -35,6 +35,11 @@ type shutdown_request =
   | Suspend
 [@@deriving rpcty]
 
+type rename_during =
```

Comment:
The type suggests the value describes a duration or period but the values suggests points in time. 
```
type rename_when = Pre_migration | Post_migration
```

Unfortunately `when` is a keyword and can't be used as a parameter name. Something like `when'` would work, though.

Reply:
`when` was also my first choice. I'll change it to see how it looks